lib/standard/stream.nit

```diff
@@ -49,18 +50,19 @@ abstract class Reader
 	fun read_byte: nullable Int is abstract
 
 	# Read at most i bytes
-	fun read(i: Int): String
+	fun read(i: Int): Bytes
```

Comment:
I can understand that the method *read i numbers of bytes* returns Bytes instead of String. In fact, it completely make sense imho; however, I do not think that it is POLA that `Reader::read` is not symmetric with `Reader::write`.

A proposal (I'm still not completely happy about it) is that you rename `Reader::read` to `Reader::read_bytes` and introduce a symmetric `Writer::write_bytes` that takes a Bytes.